amundsen_application/static/js/components/DashboardPage/index.tsx

```diff
@@ -103,6 +106,15 @@ export class DashboardPage extends React.Component<
     }
   }
 
+  onClick = (e) => {
```

Comment:
Because there are a lot of things in the page that can get clicked so in the future we could have more methods like this, let's rename this one something more specific. 

Then, to complete the functionality look for the element below where we render the group name:
```
<a
   id="dashboard-group-link"	            
   onClick={logClick}	   <--- change this from logClick to your method to test functionality, e.g. `this.onClick`           
   href={dashboard.group_url}	           
   target="_blank"	         
   rel="noopener noreferrer"	           
>	          
    {dashboard.group_name}	            
</a>	         
```